src/test/kotlin/org/dif/fixtures/Fixtures.kt

```diff
@@ -339,6 +342,24 @@ class JWM {
                      }
                 """.trimIndent(),
                 "The expected type of header 'data' is 'Map'. Got 'String'"
+            ),
+
+            WrongMessage(
+                """
+                    {
+                       "id":"1234567890",
+                       "typ":"application/didcomm-plain+json",
+                       "type":"http://example.com/protocols/lets_do_lunch/1.0/proposal",
+                       "body":{},
+                       "attachments": [{
+                            "id": "23",
+                            "data": {
+                                "base64": "qwerty\\\\"
```

Comment:
I guess it's a bug in library. We should add check for `base64` in `attachments`.

Reply:
done